packages/react-components/src/atoms/TagLabel.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import css from '@emotion/css';
+import { steel, lead } from '../colors';
+
+const borderWidth = 1;
+const containerStyles = css({
```

Comment:
Isn't this container already the molecule that contains multiple tag labels next to each other? It seems to me like this component should just have the inner element.
Also, the best tags are probably `ul > li` - I think a screen reader would otherwise read them as a sentence "Publication Preprint Paper Neurology"

Reply:
The reason to have the wrapper is  to ensure the tag always matches the content, otherwise we can have a tag stretched and it's the parent responsibility to ensure the border fits to content.

Reply:
Screenshot to represent what I'm saying.

![image](https://user-images.githubusercontent.com/763528/90245009-c1858580-de29-11ea-9895-ed6c838b3ed8.png)
![image](https://user-images.githubusercontent.com/763528/90245044-d104ce80-de29-11ea-9a0f-a9673fdf8247.png)

Reply:
It's probably not a problem anymore when we have an `ul` tab label list molecule? Then we can even stop exporting `TagLabel` from the component library to make sure it cannot be used outside of an `ul`. For now can leave it as is or try messing around with `display: inline-block` or something